### DISCUSSION TITLE
Can pikafish run muti core\threads in windows ?

### DISCUSSION THREAD

**Discussion by sum2012** [official-pikafish/Pikafish#41] [Q&A] [answered]
I mean 99% cpu times.

**Comment by sum2012** [comment]
First step I use lazurus to make a 'proxy' ucci engine that pipe with ELEEYE.EXE
[proxyucci.zip](https://github.com/official-pikafish/Pikafish/files/11830890/proxyucci.zip)

**Comment by PikaCat-OuO** [comment, marked as answer]
setoption name Threads value _your thread count_

**Comment by sum2012** [comment]
This setting seem work.Thanks anyway
![1](https://github.com/official-pikafish/Pikafish/assets/2177532/89691322-0ac9-4369-a821-4624b5adb863)
